playbooks/roles/ocp-install/tasks/main.yaml

```diff
@@ -11,11 +11,13 @@
     src: "{{ workdir }}/auth/kubeconfig"
     dest: "~/.kube/config"
 
+- include_tasks: checkcert.yaml
+
 # Run approve command till we have all workers ready. 'xargs -r ' is used to ignore empty stdin.
 - name: Approve Worker CSRs
   shell: |
     oc get csr -ojson | jq -r '.items[] | select(.status == {} ) | .metadata.name' | xargs -r oc adm certificate approve
-  until: lookup('pipe','oc get nodes | grep -w worker | grep -w Ready | wc -l') == worker_count
+  until: lookup('pipe','oc get nodes | grep -w worker | grep -w Ready | grep  -v "NotReady" | wc -l') == worker_count
```

Comment:
Any good reason for adding this? I am sure it means the same thing. However, I will go with what was working before.

Reply:
So if one goes from 4 worker nodes to three worker nodes, the get node will put the node in NotReady State.
grep -v will elude all the Nodes which are in NotReady State. This code should remain as is. I have tested going from 2 nodes to 6 nodes and back two 2 nodes over a period of 3 days.

Reply:
The previous check is `grep -w Ready` so it won't count NotReady worker nodes.

Reply:
Thanks. will do